### PR TITLE
Run3-gex38 Remove unnecessary material and add units for ZDC definition

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Remove unnecessary material and add units for ZDC definition

#### PR validation:

Test the modified run3 scenario

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cvuosalo @kpedro88 @silviodonato This is an important fix for Run3 geometry. Preferably it should be a part of the coming prerelease

**Comment by silviodonato** [comment]
urgent

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

File: Geometry/ForwardCommonData/data/zdcmaterials/2021/v1/zdcmaterials.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<DDDefinition>
+ <MaterialSection label="zdcmaterials.xml">
+  <CompositeMaterial name="Air" density="1.29*mg/cm3" symbol=" " method="mixture by weight">
+   <MaterialFraction fraction="0.7">
+    <rMaterial name="materials:Nitrogen"/>
+   </MaterialFraction>
+   <MaterialFraction fraction="0.3">
+    <rMaterial name="materials:Oxygen"/>
+   </MaterialFraction>
+  </CompositeMaterial>o
```
Comment (cvuosalo) [inline Geometry/ForwardCommonData/data/zdcmaterials/2021/v1/zdcmaterials.xml:11]: Looks like there is an "o" after `</CompositeMaterial>`. Is that valid?

Comment (bsunanda) [reply Geometry/ForwardCommonData/data/zdcmaterials/2021/v1/zdcmaterials.xml:11]: It did not create an error when i tested it. I shall fix it later. it works with that

**Pull request merged**